airbyte-e2e-testing/cypress/integration/connection.spec.js

```diff
@@ -0,0 +1,68 @@
+describe("Connection main actions", () => {
+  before(() => {
+    require('./utils.js');
+  });
+
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("Create new connection", () => {
+    cy.intercept("/api/v1/sources/check_connection").as("checkConnectionSource");
+    cy.intercept("/api/v1/destinations/check_connection").as("checkConnectionDestination");
+
+    cy.intercept("/api/v1/sources/discover_schema").as("discoverSchema");
+    cy.intercept("/api/v1/connections/create").as("createConnection");
+
+    cy.createTestSource("Test source cypress");
+    cy.createTestDestination("Test destination cypress");
+    cy.wait(3000);
+
+    cy.get("div[role=combobox]").click();
+    cy.get("div").contains("Test source cypress").click();
+    cy.wait("@checkConnectionSource");
+    cy.wait("@checkConnectionDestination");
+
+
+    cy.wait("@discoverSchema");
+
+    cy.get("div[role=combobox]").last().click();
+    cy.get("div[data-id='manual']").click();
+    cy.submit();
+
+    cy.wait("@createConnection");
+
+    cy.get("div").contains("Test source cypress").should("exist");
+    cy.get("div").contains("Test destination cypress").should("exist");
+  });
+
+  it("Update connection", () => {
```

Comment:
each test depends on the state created by the previous test? am i reading that right? is there a reasonable way to not have this be the case. we generally try to make it so that the state of each test is totally isolated from one another.